python/configuration_manager/configurations_manager.py

```diff
@@ -3,36 +3,15 @@
 #  contributor license agreements; and to You under the Apache License,
 #  Version 2.0."
 
-from definitions import CONFIG_FILE
-from xml_parser import Parser
-from config_logger import ConfigLogger
-from directories_manager import DirectoriesManager
+# TODO: relative import instead of absolute import
+from python.configuration_manager.definitions import CONFIG_FILE
```

Comment:
It looks that CONFIG_FILE is a constant whether based on Enum class or not, either case, it should be imported the package where is content and used as a part of a object of the defined Enum class.